File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        
        vector<int>merged = nums1;
        for(int i=0; i<nums2.size();i++)merged.push_back(nums2[i]);
        sort(merged.begin(),merged.end());
        if (merged.size() % 2 != 0) return static_cast<float>(merged[merged.size()/2]);
       
        else  return(static_cast<float>(merged[merged.size()/2]+merged[merged.size()/2 -1]))/static_cast<float>(2);
            
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        merged = nums1 + nums2
        merged.sort()
        if len(merged) % 2 != 0:
            return merged[len(merged)//2] 
        else:
            return (merged[(len(merged)//2)] + merged[(len(merged)//2)-1]) / 2 
        